scripts/scaffold/db/Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM postgres:13.6-alpine
+ENV POSTGRES_PASSWORD = 'example'
```

Comment:
We are starting things without `scripts/cli` bash script which actually starts this database using docker compose file `scripts/scaffold.yaml` which has all the necessary env. variables set.